model/models.py

```diff
@@ -122,9 +121,68 @@ def forward(self, sub, rel):
 		x				= self.hidden_drop2(x)
 		x				= self.bn2(x)
 		x				= F.relu(x)
-
 		x = torch.mm(x, all_ent.transpose(1,0))
 		x += self.bias.expand_as(x)
 
 		score = torch.sigmoid(x)
 		return score
+
+class CompGCN_ConvKD(CompGCNBase):
+	def __init__(self, edge_index, edge_type, params=None):
```

Comment:
Instead of params I would call it parameters, no need for abbreviations

Reply:
I would keep the name 'params' since the other scoring functions also call it like that. Just to be uniform